### PR TITLE
Update notch options

### PR DESCRIPTION
**Pull request by Hwurzburg**
added options and explanations

### CODE REVIEW

File: common/source/docs/common-imu-notch-filtering.rst [hunk 1]
```diff
@@ -67,13 +67,25 @@ Checking Notch Filter Effectiveness
...
+The software notch filters used are very "spikey" being relatively narrow but good at attenuation at their center. On larger copters the noise profile of the motors is quite dirty covering a broader range of frequencies than can be covered by a single notch filter. In order to address this situation it is possible to configure the harmonic notches as multiple notches that gives a wider spread of significant attenuation. The configuration is controlled by the :ref:`INS_HNTCH_OPTS <INS_HNTCH_OPTS>` parameter. This is a bitmask parameter and multiple options are possible at the same time, but using bit 0, 1, or bit 4 at the same time should be avoided.
+
+==========================================      =======================
+:ref:`INS_HNTCH_OPTS <INS_HNTCH_OPTS>` Bit      Action
+==========================================      =======================
+0                                               Double side by side Notches
```
Comment (andyp1per) [inline common/source/docs/common-imu-notch-filtering.rst:78]: Actually both double and triple notches are not side-by-side, but overlapping. Overlapping means that there is still significant attenuation between the notches

File: common/source/docs/common-imu-notch-filtering.rst [hunk 2]
```diff
@@ -67,13 +67,25 @@ Checking Notch Filter Effectiveness
...
+1                                               MultiSource: if using FFT Mode, the three largest noise sources will have a notch assigned. If ESC Telemetry Mode, then each motor will have a notch assigned at its RPM.
+2                                               Updates the filters at the loop rate. This is cpu intensive, but tracks noise variations faster.
+3                                               Enables notches on every IMU instead of just the primary. This is cpu intensive, but allows better lane switching decisions in noisy situations
```
Comment (andyp1per) [inline common/source/docs/common-imu-notch-filtering.rst:81]: Also good for debug, on H7 its fine to set this. On F4 not so much.

File: common/source/docs/common-imu-notch-filtering.rst [hunk 3]
```diff
@@ -67,13 +67,25 @@ Checking Notch Filter Effectiveness
...
+
+==========================================      =======================
+:ref:`INS_HNTCH_OPTS <INS_HNTCH_OPTS>` Bit      Action
+==========================================      =======================
+0                                               Double side by side Notches
+1                                               MultiSource: if using FFT Mode, the three largest noise sources will have a notch assigned. If ESC Telemetry Mode, then each motor will have a notch assigned at its RPM.
+2                                               Updates the filters at the loop rate. This is cpu intensive, but tracks noise variations faster.
```
Comment (andyp1per) [inline common/source/docs/common-imu-notch-filtering.rst:80]: Its only really worth it if you have an update source that is fast - e.g. bdshot

File: common/source/docs/common-imu-notch-filtering.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-imu-notch-filtering.rst:78]: changed

File: common/source/docs/common-imu-notch-filtering.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-imu-notch-filtering.rst:81]: added

**Pull request merged**